ui/jobs/jobs.js

```diff
@@ -1618,44 +1620,107 @@ class Bars {
     };
   }
 
-  // TODO: none of this is actually super useful.
   setupAst() {
+    const cardsMap = {
+      'Balance': { 'bonus': 'melee', 'seal': 'Solar' },
+      'Bole': { 'bonus': 'range', 'seal': 'Solar' },
+      'Arrow': { 'bonus': 'melee', 'seal': 'Lunar' },
+      'Ewer': { 'bonus': 'range', 'seal': 'Lunar' },
+      'Spear': { 'bonus': 'melee', 'seal': 'Celestial' },
+      'Spire': { 'bonus': 'range', 'seal': 'Celestial' },
+    };
+
     let combustBox = this.addProcBox({
       id: 'ast-procs-combust',
       fgColor: 'ast-color-combust',
     });
 
-    let beneficBox = this.addProcBox({
-      id: 'ast-procs-benefic',
-      fgColor: 'ast-color-benefic',
+    let drawBox = this.addProcBox({
+      id: 'ast-procs-draw',
+      fgColor: 'ast-color-draw',
+    });
+
+    let lucidBox = this.addProcBox({
+      id: 'ast-procs-luciddreaming',
+      fgColor: 'ast-color-lucid',
+    });
+
+    let cardBox = this.addResourceBox({
+      classList: ['ast-color-card'],
+    });
+
+    let sealBox = this.addResourceBox({
+      classList: ['ast-color-seal'],
     });
 
-    let heliosBox = this.addProcBox({
-      id: 'ast-procs-helios',
-      fgColor: 'ast-color-helios',
+    this.jobFuncs.push((jobDetail) => {
+      let card = jobDetail.heldCard;
+      let seals = jobDetail.arcanums;
```

Comment:
Are these variables declared as `const` better?
I think they couldn't be changed theoretically.